mmocr/models/textdet/necks/fpn_cat.py

```diff
@@ -135,9 +157,111 @@ def forward(self, inputs):
         for i, out in enumerate(outs):
             outs[i] = F.interpolate(
                 outs[i], size=outs[0].shape[2:], mode='nearest')
+
         out = torch.cat(outs, dim=1)
+        if self.asf_cfg is not None:
+            asf_feature = self.asf_conv(out)
+            attention = self.asf_attn(asf_feature)
+            enhanced_feature = []
+            for i, out in enumerate(outs):
+                enhanced_feature.append(attention[:, i:i + 1] * outs[i])
+            out = torch.cat(enhanced_feature, dim=1)
 
         if self.conv_after_concat:
             out = self.out_conv(out)
 
         return out
+
+
+class ScaleChannelSpatialAttention(BaseModule):
+    """Spatial Attention module in Real-Time Scene Text Detection with
+    Differentiable Binarization and Adaptive Scale Fusion.
+
+    This was partially adapted from https://github.com/MhLiao/DB
+
+    Args:
+        in_channels (int): A numbers of input channels.
+        c_wise_channels (int): Number of channel-wise attention channels.
+        out_channels (int): Number of output channels.
+        init_cfg (dict or list[dict], optional): Initialization configs.
+    """
+
+    def __init__(self,
+                 in_channels,
+                 c_wise_channels,
+                 out_channels,
+                 init_cfg=None):
```

Comment:
```suggestion
                 init_cfg=[dict(type='Kaiming', layer='Conv', bias=0), dict(type='Constant', layer='BatchNorm', val=1., bias=0)]):
```
Missing weight initialization configs. https://github.com/MhLiao/DB/blob/760d9217ac252140ab65410c0fcc53b2d4a0cf80/decoders/feature_attention.py#L58-L66